### PR TITLE
fix(gatsby docs): add x-transport on servers to default inspector transport

### PR DESCRIPTION
**Pull request by shanejonas**
image

image

### CODE REVIEW

File: templates/docs/gatsby/src/pages/api-documentation.tsx [hunk 1]
```diff
@@ -70,8 +71,14 @@ const ApiDocumentation: React.FC = () => {
...
 
   useEffect(() => {
-    if (openrpcDocument && openrpcDocument.servers && openrpcDocument.servers[0]) {
+    if (!openrpcDocument) {
+      return;
+    }
+    if (openrpcDocument.servers && openrpcDocument.servers[0]) {
       setInspectorUrl(openrpcDocument.servers[0].url);
+      if (openrpcDocument.servers[0]["x-transport"]) {
+        setInspectorTransport(openrpcDocument.servers[0]["x-transport"]);
```
Comment (BelfordZ) [inline templates/docs/gatsby/src/pages/api-documentation.tsx:80]: maybe we should make a generic-ish component for 'server picker' ?

Comment (shanejonas) [reply templates/docs/gatsby/src/pages/api-documentation.tsx:80]: yea we definitely should. Dunno if it should block this PR though.

Comment (shanejonas) [reply templates/docs/gatsby/src/pages/api-documentation.tsx:80]: created #484 to track it

**Pull request merged**